docs/api.adoc

```diff
@@ -328,6 +328,9 @@ image:images/v-500.svg[Since version,opts=inline] Represents result of HtmlArea
 |images: [<<Image>>]
 |Represents details of images which were detected in the HtmlArea value, except images with `original` styles
 
+|links: [<<Link>>]
+|Represents details of links which were detected in the HtmlArea value, where value the `href` attribute starts from `content://`, `media://` or `image://`
```

Comment:
image too?
Actually it is better to rephrase it, so we don't mention uri scheme,  but explain what they represent (content or media)